core/tests/test_utils.py

```diff
@@ -510,18 +530,51 @@ def get_response(self, url, params=None, expect_errors=False,
 
         return response
 
+    def get_html_response(self, url, params=None, expected_status_int=200):
+        """Get a HTML response, transformed to a Python object.
+
+        Args:
+            url: str. The URL to fetch the response.
+
+            params: str or dict. A query string, or a dictionary that will be
```

Comment:
See above notes on how to write docstrings and on the idea of making the expected usage of params more explicit.

Reply:
done